src/targets/services/konnectorAlerts/destroyTriggerAt.js

```diff
@@ -0,0 +1,28 @@
+import { logger } from 'ducks/konnectorAlerts'
+import { getTriggerStates, fetchRelatedAtTriggers } from './helpers'
+
+export const destroyTriggerAt = async client => {
+  logger('info', `⌛ Try to destroy @at triggers...`)
+
+  const count = []
+
+  const settingTriggerStates = await getTriggerStates(client)
+
+  for (const [id, triggerStates] of Object.entries(settingTriggerStates)) {
+    const relatedAtTriggers = await fetchRelatedAtTriggers(client, id)
```

Comment:
C'est pas terrible de faire du fetch dans une boucle. On peut pas récupérer tous les IDs en amont et faire une requête pour tout récupérer d'un coup ?

Reply:
D'ailleurs ne devrait on pas exclure en amont les status !== 'errored' histoire de ne pas récupérer des ID qu'on n'utilisera pas ?

Reply:
Tout à fait c'est ce que je m'étais dit mais ça m'est sorti de la tête :(

`fetchRelatedAtTriggers` fait un `Q(TRIGGER_DOCTYPE).where({ type: '@at', 'message.konnectorTriggerId': id })` on pourrait probablement dans le where faire quelque chose genre `{_id: "$in": ids}`